.golangci.yml

```diff
@@ -4,30 +4,35 @@ run:
   timeout: 5m
   allow-parallel-runners: true
 
-formatters:
-  enable:
-  - goimports
```

Comment:
Please revert: `goimports` formatter is required. When missing the `formatters` section only 'gofmt` is run.